packages/odyssey-react/src/components/VisuallyHidden/index.tsx

```diff
@@ -0,0 +1,41 @@
+/*!
+ * Copyright (c) 2021-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import type { FunctionComponent, ReactText } from 'react';
+import { useOmit } from '../../utils';
+import styles from './VisuallyHidden.module.scss';
+
+export type Props = {
+  /**
+   * Text content to be read only to screen readers
+   */
+  children: ReactText,
+};
+
+const VisuallyHidden: FunctionComponent<Props> = (props) => {
```

Comment:
two things - first what do you think about naming this for what it is, rather than what it is not? That is, rather than saying it is not visible (i.e. hidden) leaving the intended functionality ambiguous, name it `ScreenReaderText` or similar, making the purpose explicit. This is how your PR description reads as well!

Second it would be nice to use a more sophisticated type annotation here to make clear the props such as `style` and `className` are not accepted. Something like this:

```
export interface Props extends Omit<
  ComponentPropsWithoutRef<'span'>,
  'style' | 'className'
> {
    children: ReactText
}


// and then here:

const ScreenReaderOnlyText = (props: Props) => {
}
```

Reply:
Updated on both counts!